qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/VisitSteps.java

```diff
@@ -121,5 +134,71 @@ public void clickOnTheEndVisitButton() {
     @Then("the system ends the visit")
     public void systemEndsVisit() {
     	assertTrue(textExists("No active visit"));
-    }   
+    }
+
+    @When("user clicks on the Active Visits app from the home page")
+    public void loadActiveVisitsPage() {
+    	activeVisitsPage = homePage.goToActiveVisitsSearch();
+    }
+
+    @And("user searches for the patient with active visit using patient Identifier")
+    public void searchForPatientUsingPatientIdentifier() {
+    	activeVisitsPage.search(patient.identifier);
+    	activeVisitsPage.waitForPageToLoad();
+    }
+
+    @And("the system returns patient with the provided patient Identifier")
+    public void systemReturnsPatientWithProvidedIdentifier() {
+    	assertThat(activeVisitsPage.getPatientIdOfLastActiveVisit(), containsString(patient.identifier));
+    }
+
+    @And("user clicks on the patient returned with active visit")
+    public void loadPatientDashboardPage() {
+    	dashboardPage = activeVisitsPage.goToPatientDashboardOfLastActiveVisit();
+    }
+
+    @Then("the system loads the patient dashboard")
+    public void systemLoadsPatientDashboard() {
+    	assertPage(dashboardPage.waitForPage());
+    	dashboardPage.goToHomePage();
+    }
+
+    @And("user searches for the patient with active visit using patient Id")
+    public void searchForPatientUsingPatientId() {
+    	String PATIENT_ID = activeVisitsPage.getPatientIdOfLastActiveVisit();
```

Comment:
Any reason why upper-casing the variable name?

Reply:
> Any reason why upper-casing the variable name?

@kdaud  no it was an oversight, I had initially tried making them constants at the top hence the caps. Let me fix this.

Reply:
Oh I see! the blind man told his deaf wife!

Reply:
@kdaud  I have pushed a commit. You could make more reviews.